spring-cloud-dataflow-server-core/src/main/resources/schema-hsqldb.sql

```diff
@@ -1,6 +1,15 @@
 
-CREATE TABLE TASK_DEFINITION  (
+CREATE TABLE TASK_DEFINITIONS  (
 	DEFINITION_NAME VARCHAR(255) NOT NULL PRIMARY KEY,
 	DEFINITION CLOB DEFAULT NULL
 );
 
+CREATE TABLE STREAM_DEFINITIONS  (
+	DEFINITION_NAME VARCHAR(255) NOT NULL PRIMARY KEY,
+	DEFINITION CLOB DEFAULT NULL
+);
+
+CREATE TABLE DEPLOYMENT_IDS  (
+	DEPLOYMENT_KEY VARCHAR(255) NOT NULL PRIMARY KEY,
+	DEPLOYMENT_ID VARCHAR(255) NOT NULL
+);
```

Comment:
Well... I think there is a problem.  Before we could do the following:
task create foo --definition "timestamp" 
task launch foo
task launch foo
Now we get an exception on the 2nd launch.  
We can put a try catch here: https://github.com/spring-cloud/spring-cloud-dataflow/blob/e03a2db23d67190ff95d2702e1dc911e18761c4e/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskService.java#L166   
to ignore that exception.